src/Feed/EventDatabaseApiFeedType.php

```diff
@@ -54,9 +55,9 @@ public function getData(Feed $feed): array
 
                         $queryParams = array_filter([
                             'items_per_page' => $numberOfItems,
-                            'occurrences.place.id' => $places,
-                            'organizer.id' => $organizers,
-                            'tags' => $tags,
+                            'occurrences.place.id' => array_map(function ($place) { return str_replace('/api/places/', '', $place['value']); }, $places),
+                            'organizer.id' => array_map(function ($organizer) { return str_replace('/api/organizers/', '', $organizer['value']); }, $organizers),
+                            'tags' => array_map(function ($tag) { return str_replace('/api/tags/', '', $tag['value']); }, $tags),
```

Comment:
https://www.php.net/manual/en/functions.arrow.php are designed for this. Consider using them, i.e.

```suggestion
                            'occurrences.place.id' => array_map(static fn($place) => str_replace('/api/places/', '', $place['value']), $places),
                            'organizer.id' => array_map(static fn($organizer) => str_replace('/api/organizers/', '', $organizer['value']), $organizers),
                            'tags' => array_map(static fn($tag) => str_replace('/api/tags/', '', $tag['value']), $tags),
```